lib/game_teams_collection.rb

```diff
@@ -13,6 +13,7 @@ def initialize(path)
   end
 
   def from_csv(path)
+    return if !File.exist?(path)
```

Comment:
I believe the ||= syntax will also do this, we can do some refactoring here if needed